### DISCUSSION TITLE
In State GST Display Issue

### DISCUSSION THREAD

**Discussion by amet123** [frappe/print_designer#233] [Q&A] [answered]
Hi 
there is no option to show In State GST Breakup to Tax section

Thanks

**Comment by maharshivpatel** [comment]
@amet123 just create dynamic text field, double click to open the dialog. then enable hidden field toggle and you should be able to see taxes and charges field

**Comment by amet123** [comment]
@maharshivpatel  thanks for reply ,  Only out-State GST Taxs show,  when you select In-State GST nothing will show in print
means CGST 9% -SGST 9%

**Comment by maharshivpatel** [comment]
@amet123 can you share screenshot? also, update to latest version of print_designer and save the format once.

**Comment by amet123** [comment]
@maharshivpatel  ![image](https://github.com/frappe/print_designer/assets/326649/51d341f3-5bad-4c1c-b445-1220c1faad84)

Print Designer: v1.0.0 (main)

Pls Look @ this

**Comment by maharshivpatel** [comment, marked as answer]
@amet123 As taxes are saved in the child table. you need to create a table, select `Sales Taxes and Charges` and remove the padding, and header to get this look.


<img width="1438" alt="Screenshot 2024-04-08 at 3 14 36 PM" src="https://github.com/frappe/print_designer/assets/39730881/0a0d81b0-9d6d-4324-8de1-5a89616da8f8">

**Reply by amet123** [reply]
Hi @maharshivpatel 

Sorry to say after update unable to find SGST in after enable hidden fields  (in table , Select sales Taxes and chart)

in quotation and sales order
